Cargo.toml

```diff
@@ -30,13 +30,14 @@ clap = { version = "4", features = ["derive"] }
 config = "0.13.3"
 env_logger = "0.10.0"
 hex = "0.4.3"
-kbs-types = "0.5.3"
+kbs-types = "0.6.0"
 jsonwebtoken = { version = "9", default-features = false }
 log = "0.4.17"
 prost = "0.11.0"
 regorus = { version = "0.1.5", default-features = false, features = ["regex", "base64", "time"] }
 rstest = "0.18.1"
 serde = { version = "1.0", features = ["derive"] }
+serde_with = { version = "1.11.0", features = ["base64"] }
```

Comment:
Let's put `serde_with` after `serde_json` to follow dictionary order

Reply:
Fixed, thanks!